### PR TITLE
Added support of .env file for standalone scripts (e.g. taoInstall.php).

### PR DESCRIPTION
**Pull request by julien-sebire**
This PR enables reading of optional `.env` file in project root directory (see https://symfony.com/doc/current/components/dotenv.html for more details) in all standalone scripts requiring `tao/install/init.php`.
The same may be done for other parts of the application. .env is already available for the application as soon as it is bootstraped.

### CODE REVIEW

File: install/init.php [hunk 1]
```diff
@@ -30,7 +33,13 @@
 require_once ($root . 'vendor/autoload.php');
 
 if (tao_install_utils_System::isTAOInstalled()) {
-    require_once ($root . '/config/generis.conf.php');
+    require_once ($root . 'config/generis.conf.php');
+}
+
+$envFile = $root . '.env';
+if (file_exists($envFile)) {
+    $dotenv = new Dotenv();
```
Comment (boajer) [inline install/init.php]: That is the 3rd place in the platform with env file loading. Might be we should reduce code duplication by wrapping it into class?

Comment (peetya) [reply install/init.php]: Agree. I'm not familiar with the infrastructure, but in best-case scenario we should define it only once. If TAO has multiple entrypoints (for instance web, cli, worker, etc...) we need to make a reusable class.

File: install/init.php [hunk 2]
```diff
@@ -30,9 +33,11 @@
 require_once ($root . 'vendor/autoload.php');
 
 if (tao_install_utils_System::isTAOInstalled()) {
-    require_once ($root . '/config/generis.conf.php');
+    require_once ($root . 'config/generis.conf.php');
```
Comment (mike-ionut-mihai-sandu-tao) [inline install/init.php]: why did you remove the leading `/`

File: install/init.php [hunk 3]
```diff
@@ -19,6 +19,9 @@
  *               2009-2012 (update and modification) Public Research Centre Henri Tudor (under the project TAO-SUSTAIN & TAO-DEV);
  *               2013-     (update and modification) Open Assessment Technologies SA;
  */
+
+use oat\tao\model\mvc\DotEnvReader;
```
Comment (mike-ionut-mihai-sandu-tao) [inline install/init.php]: should we use the service locator for this?

File: manifest.php [hunk 4]
```diff
@@ -54,7 +54,7 @@
     'label' => 'TAO Base',
     'description' => 'TAO meta-extension',
     'license' => 'GPL-2.0',
-    'version' => '39.5.2',
+    'version' => '39.6.0',
```
Comment (mike-ionut-mihai-sandu-tao) [inline manifest.php]: why is this a major?

Comment (julien-sebire) [reply manifest.php]: Afaik, in semver:
- Backward compatible breaking change => major
- New feature => minor
- Bug fix => release

File: install/init.php (hunk 2 above)
Comment (julien-sebire) [reply install/init.php]: It's trailing in the `$root` variable already. This was doubling the `/`.

File: install/init.php (hunk 3 above)
Comment (julien-sebire) [reply install/init.php]: I don't think so. This is very static indeed and has no effect if the file is not present.

File: models/classes/mvc/DotEnvReader.php [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+{
+    /**
+     * Reads .env file into $_ENV.
+     *
+     * @param string $envFile (defaults to project .env file)
+     */
+    public function __construct($envFile = '')
+    {
+        if ($envFile === '') {
+            $envFile = __DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR
```
Comment (martijn-tao) [inline models/classes/mvc/DotEnvReader.php]: To make perhaps make this a bit more readable, you could do something like:

```
__DIR__  . implode(DIRECTORY_SEPARATOR, array_fill(1, 4, '..')) . '.env'
```

File: install/init.php (hunk 2 above)
Comment (hutnikau) [reply install/init.php]: Please use `DIRECTORY_SEPARATOR` constant in paths.

**Comment by mike-ionut-mihai-sandu-tao** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
